datajoint/hub.py

```diff
@@ -0,0 +1,43 @@
+import re
+import requests
+from requests.adapters import HTTPAdapter
+from datajoint.errors import DataJointError
+
+HUB_PROTOCOL = 'hub://'
+REQUEST_PROTOCOL = 'https://'
+API_ROUTE = '/api'
+API_TARGETS = dict(GET_DB_FQDN='/v0.000/get_database_fqdn')
```

Comment:
Let's change to /projectdatabaseinstance/get to make it more 'resty'

Reply:
@ixcat Actually, if the aim is to make this more 'RESTy', we should likely modify this from a `POST` to a `GET` since we are performing a 'read-only' operation. Usually, the method dictates the operation. If parameters need to be supplied, they can be passed as query params e.g. a `GET` on `/v0.000/projectdatabaseinstance?org_name=datajoint&project_name=travis`. If you agree, I can make the updates.

Reply:
Yes - this is true. Some background on why this was implemented this way is that we had problems in IBL for 'subsets' in using a pure GET/'HTTP Verb' approach since it wasn't possible to construct >1x values to filter a query request for e.g. both 'project1' and 'project2' against the 'root' node (GET parameters get squashed - e.g. 2x 'project=X' will only pass one value to the handler) and this doesn't flatten to a clean URL either.

In those cases we set a single value in `_json` post variable which was then decoded to perform the N-valued restriction This doesn't directly apply here since we are targeting only one object at a time, but that usage might come in handy later (e.g. 'show me the databases for this subset of projects').. not sure what's best as an overall approach. A middle ground might be to offer fallback to a post queries for multi-valued filtering and allow GET filtering for the normal case. One difference from IBL worth noting for the discussion is that this service will likely grow to be read-write, or at least some other similar API will grow to be read-write, so we'll need to find a suitable mechanism for read-write cases either way.

Pinging @eywalker for feedback